src/Symfony/Component/Uid/Uuid.php

```diff
@@ -100,27 +95,23 @@ public function getType(): int
         return uuid_type($this->uuid);
     }
 
-    public function getVariant(): int
-    {
-        return uuid_variant($this->uuid);
-    }
-
-    public function getTime(): int
+    public function getTime(): float
     {
         if (self::TYPE_1 !== $t = uuid_type($this->uuid)) {
             throw new \LogicException("UUID of type $t doesn't contain a time.");
         }
 
-        return uuid_time($this->uuid);
-    }
+        $time = '0'.substr($this->uuid, 15, 3).substr($this->uuid, 9, 4).substr($this->uuid, 0, 8);
 
-    public function getMac(): string
```

Comment:
Same as getVariant: nobody needs this IMHO - ppl that do can still use `uuid_mac()`